views/email/email_partials/email_footer.hbs

```diff
@@ -23,7 +23,7 @@
       <p style="padding: 0px 0px 3px 0px; margin: 0px; font-weight: 600; color: #000000;">Mozilla</p>
       <p style="padding: 0px 0px 3px 0px; margin: 0px; font-weight: 300; color: #5d5d5d;">331 E. Evelyn Avenue Mountain View CA 94041</p>
       <p style="padding: 0px; margin: 0px;">
-        <a href="https://www.mozilla.org/about/legal/" rel="noopener" style="text-decoration: none; color: #0060df;" >Legal</a> | <a href="https://www.mozilla.org/privacy/websites/" rel="noopener" style="text-decoration: none; color: #0060df;">Privacy</a>
+        <a href="https://www.mozilla.org/about/legal/?utm_source=email&utm_medium=email&utm_campaign=fx_monitor_downloads&content=email_footer" rel="noopener" style="text-decoration: none; color: #0060df;" >Legal</a> | <a href="https://www.mozilla.org/privacy/websites/?utm_source=email&utm_medium=email&utm_campaign=fx_monitor_downloads&content=email_footer" rel="noopener" style="text-decoration: none; color: #0060df;">Privacy</a>
```

Comment:
Probably overthinking it, but would it make sense to put these into a HBS variable?

```hbs
<a href="https://www.mozilla.org/about/legal/?{{ utm_email_footer }}" rel="noopener" ...>
```

Reply:
Good idea. HBS doesn't support variable assignments in templates themselves, and it feels like a mis-match to put this in the controllers.

I'm going to try to move these into partials.

Reply:
Sure, I could see a partial or in server.js along w/ other `app.locals.BLAH` scoped stuff:

```js
app.locals.UTM_SOURCE = url.parse(AppConstants.SERVER_URL).hostname;
app.locals.UTM_EMAIL_FOOTER = "?utm_source=email&utm_medium=email&utm_campaign=fx_monitor_downloads&utm_content=email-footer-link";
app.locals.UTM_SITE_FOOTER = `?utm_source=${app.locals.UTM_SOURCE}&utm_medium=referral&utm_campaign=fx_monitor_downloads&utm_content=site-footer-link`;
```

```hbs
<a class="footer-logo-wrapper" href="https://www.mozilla.org/{{ UTM_SITE_FOOTER }}" target="_blank" rel="noopener">...</a>

<a rel="noopener" href="https://www.mozilla.org/{{ UTM_EMAIL_FOOTER }}" style="margin: 0px; padding: 0px;">...</a>
```

PROS:
- 2 fewer partials
- keeps UTM_SOURCE and other UTM_* stuff a bit closer together

CONS:
- polluting app.locals.*